src/Database/LSMTree/Monoidal.hs

```diff
@@ -681,7 +672,7 @@ union :: forall m k v.
   => Table m k v
   -> Table m k v
   -> m (Table m k v)
-union = undefined
+union = error "union: not yet implemented" $ union @m @k @v
```

Comment:
So the recursive call avoids redundant constraint warnings?

Reply:
Yes, exactly